src/authentification/components/AuthRegisterForm.tsx

```diff
@@ -17,10 +17,22 @@ export default function AuthRegisterForm() {
 
   const form = useForm<User>();
   const [userData, setUserData] = useRecoilState(userState);
+  const [isLoggedIn, setIsLoggedIn] = useRecoilState(loggedState);
 
   const onSubmit = (data: User) => {
     createUserWithEmailAndPassword(auth, data.email, data.password)
-      .then(() => {
+      .then(async (userCredential) => {
+        if (auth.currentUser) {
```

Comment:
It's better to check `userCredentials` here that the request returns then `auth.currentUser`

Reply:
Added check for userCredentials and auth.currentUser.